test/e2e/operation/pipeline/src/test/java/org/apache/shardingsphere/test/e2e/data/pipeline/framework/container/compose/DockerContainerComposer.java

```diff
@@ -66,15 +67,18 @@ public DockerContainerComposer(final DatabaseType databaseType, final String sto
         }
         for (int i = 0; i < storageContainerCount; i++) {
             StorageContainerConfiguration storageContainerConfig;
+            int majorVersion = new DockerImageVersion(storageContainerImage).getMajorVersion();
             if (DatabaseTypeUtils.isMySQL(databaseType)) {
-                int majorVersion = new DockerImageVersion(storageContainerImage).getMajorVersion();
                 Map<String, String> mountedResources = Collections.singletonMap(String.format("/env/mysql/mysql%s/my.cnf", majorVersion), MySQLContainer.MYSQL_CONF_IN_CONTAINER);
                 storageContainerConfig = MySQLContainerConfigurationFactory.newInstance(null, null, mountedResources);
             } else {
                 storageContainerConfig = StorageContainerConfigurationFactory.newInstance(databaseType);
             }
             DockerStorageContainer storageContainer = getContainers().registerContainer((DockerStorageContainer) StorageContainerFactory.newInstance(databaseType, storageContainerImage, null,
                     storageContainerConfig));
+            if (majorVersion > 5 && DatabaseTypeUtils.isMySQL(databaseType)) {
+                storageContainer.withClasspathResourceMapping("/env/mysql/mysql8/02-initdb.sql", "/docker-entrypoint-initdb.d/02-initdb.sql", BindMode.READ_ONLY);
+            }
```

Comment:
There're duplicated `DatabaseTypeUtils.isMySQL(databaseType)`, could we merge these code blocks for better readability?